packages/backend/src/handlers/actions/3Box/refresh3BoxCache/cache3BoxProfiles.ts

```diff
@@ -0,0 +1,16 @@
+import { CacheProcessOutput } from '../../../../lib/autogen/hasura-sdk';
+import { client } from '../../../../lib/hasuraClient';
+import { updateCachedProfile } from '../cache3BoxProfile/updateCachedProfile';
+
+export async function cache3BoxProfiles(): Promise<CacheProcessOutput> {
```

Comment:
the function naming and folder structure is kinda confusing. cache3BoxProfiles function is not inside the cache3BoxProfile folder. Seems like they should all be in the same directory.

Reply:
There are folders for each of the action handlers. The update implementation is in the same directory as those folders.